Deploy-CAM.ps1

```diff
@@ -3558,16 +3587,12 @@ $selectedSubcriptionId = $subscriptionsToDisplay[$chosenSubscriptionIndex].Subsc
 
 # Now we have the subscription set. Ensure it has keyvault resource provider
 $keyVaultProviderExists = [bool](Get-AzureRmResourceProvider | Where-Object {$_.ProviderNamespace -eq "Microsoft.Keyvault"})
```

Comment:
Is this line still needed?

Reply:
Nope, was from the previous code, not sure why me from a week ago didn't remove it. Updated.